src/index.ts

```diff
@@ -0,0 +1,17 @@
+import express from 'express';
+import initialize from './app';
+import { connect } from './connectToMongo';
+
+const { MONGO_CONNECTION_STRING } = process.env as {
+  [key: string]: string;
+};
+
+connect(MONGO_CONNECTION_STRING);
+
+const app = express();
```

Comment:
I'm curious we need this variable here. Could we move it to method initialize? Then we can remove additional parameter in this method

Reply:
Moved to initialize